src/test/java/UtilTest.java

```diff
@@ -0,0 +1,62 @@
+import SootUp.Util;
+import com.googlecode.dex2jar.tools.BaseCmd;
+import org.junit.jupiter.api.Test;
+import sootup.core.model.SootClass;
+import sootup.core.model.SootField;
+import sootup.core.model.SootMethod;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UtilTest {
+
+    @Test
+    public void testLoadClass() {
+        Optional<SootClass> realClass = Util.loadClass("Test");
```

Comment:
Meinem bisherigen Verständnis nach sind die Inhalte des `demo` Verzeichnisses recht variabel. Dann sollten wir nicht unsere Tests davon abhängig machen, sondern sie in einen anderen Ordner schieben.